scripts/bundle/bundle-linux.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+PORTER_MIRROR=${PORTER_MIRROR:-https://cdn.porter.sh}
+PORTER_VERSION=${PORTER_VERSION:-latest}
+MONGO_IMAGE_FILE="mongo_image.tar"
+MONGO_IMAGE_URL="docker.io/library/mongo:4.0-xenial"
+BUNDLE_NAME="porter-air-gapped-install-$PORTER_VERSION"
+DOWNLOAD_DIR="/tmp/$BUNDLE_NAME"
+
+PORTER_BINARY_URL=${PORTER_BINARY_URL:-"$PORTER_MIRROR/$PORTER_VERSION/porter-linux-amd64"}
+PORTER_SCRIPT_URL=${PORTER_SCRIPT_URL:-"$PORTER_MIRROR/$PORTER_VERSION/install-linux.sh"}
+
+mkdir -p $DOWNLOAD_DIR
+
+echo "Pulling MongoDB image..."
+docker pull "$MONGO_IMAGE_URL"
+echo "Saving MongoDB image to $MONGO_IMAGE_FILE..."
+docker save -o "$DOWNLOAD_DIR/$MONGO_IMAGE_FILE" "$MONGO_IMAGE_URL"
+
+echo "Downloading Porter Install Script from $PORTER_SCRIPT_URL"
+curl -fsSLo "$DOWNLOAD_DIR/install-linux.sh" "$PORTER_SCRIPT_URL"
+
+echo "Downloading Porter Binary from $PORTER_BINARY_URL"
+curl -fsSLo "$DOWNLOAD_DIR/porter-linux-amd64" "$PORTER_BINARY_URL" 
+
+cat << EOF > $DOWNLOAD_DIR/install-bundle.sh 
+export PORTER_BINARY_URL="file://\$(realpath ./porter-linux-amd64)" 
+export PORTER_SCRIPT_URL="file://\$(realpath ./install-linux.sh)" 
+bash install-linux.sh
+docker load -i "$MONGO_IMAGE_FILE"
+if ! docker ps -q -f name=porter-mongodb-docker-plugin; then
+  echo "Running MongoDB container..."
+  docker run --name porter-mongodb-docker-plugin -d -p 27018:27017 -v mongodb_data:/data/db --restart always mongo:4.0-xenial
```

Comment:
Any specific reason for starting MongoDB during installation?

Reply:
Yes, this specifically addresses issue #3198.

Reply:
Do you think it would make sense to mention that it starts MongoDB in the documentation? It feels a little unexpected.
Side question: If MongoDB is not running next time porter is run, will it work when the container have been started once, or will it require a manual start of the container?

Reply:
A manual restart may be required, but only if the docker container is stopped or destroyed. Mongo is deployed with `--restart always` so hopefully it won't be an issue that arises that often.

Reply:
I'll update the docs to explain mongo, also I'll see what the behavior is currently on `main` branch when you stop mongo and run porter a second time.

Reply:
@jmcudd I know we all are busy, but was wondering if you had any update on this?